apps/src/aiTutor/views/WarningModal.tsx

```diff
@@ -35,37 +28,7 @@ const WarningModal = () => {
     return null;
   }
 
-  return (
-    <AccessibleDialog onClose={onClose} className={style.chatWarningModal}>
-      <Heading2>Remember to chat responsibly!</Heading2>
-
-      <button type="button" onClick={onClose} className={style.xCloseButton}>
-        <i id="x-close" className="fa-solid fa-xmark" />
-      </button>
-      <hr />
-      <BodyTwoText>
-        <StrongText>All messages sent to AI Tutor are recorded</StrongText>
```

Comment:
Looks like there are small copy differences between the two (eg, using AI Tutor instead of AI Chat, not sure if there are others?)

Reply:
Yeah, also "Ask AI Tutor" -> "Start chatting" on the button. I think it's ultimately a Product question if we want to streamline or customize strings @samantha-code Do you have a preference?

Reply:
I think we should streamline unless there is a really obvious need to be different which I dont see here.

Reply:
Streamlined! To make them more generic, I removed references to the specific tooling. 
![Screenshot 2024-06-07 at 12 40 40 PM](https://github.com/code-dot-org/code-dot-org/assets/12300669/0cb69373-1429-4673-89e4-af944d8c3150)